src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -43,6 +42,8 @@ public sealed class Cookie
         internal const int MaxSupportedVersion = 1;
         internal const string MaxSupportedVersionString = "1";
 
+        internal const string NameAttributeName = "Name";
+        internal const string ValueAttributeName = "Value";
```

Comment:
This PR isn't just a test PR, it's also a code cleanup one. 

I think the change is too small to require it's own commit and matches the code cleanup part of the PR

I can change the description and title of the Commit however to reflect what's changed